dash/frontend/src/app/modules/private/pages/change-password/change-password.component.html

```diff
@@ -3,7 +3,7 @@
     <mat-card appearance="outlined" class="change-password-mat-card">
       <mat-card-header>
         <mat-card-title>
-          <h3>Change Password</h3>
+          <a>Change Password</a>
```

Comment:
Is an <a> tag the right element for this?

Reply:
The document refers to the link in the menu. The menu's "Change password" should be a link.
![image](https://github.com/m9sweeper/m9sweeper/assets/50845926/d6ae2866-74cb-4436-92e8-024c5d7d5e2e)

Reply:
Change Password is already a link to /private/change-password.

Reply:
It isn't - it's a button. It redirects to /private/change-password, but it isn't a link.

Reply:
@becky-intelletive The items on this dropdown menu are all buttons. Help me to understand the thinking behind changing this from a button to a link.

Reply:
It's a link, so it should act like one.
You can make it a button/link situation like the documentation maybe? Would that make more sense?